src/pages/ThorChainLP/AvailablePools.tsx

```diff
@@ -121,60 +112,32 @@ export const AvailablePools = () => {
       },
       {
         Header: translate('pools.volume24h'),
-        // TODO(gomes): expose me in the Pool interface so this can be used for sorting
-        // accessor: 'volume24h',
+        accessor: 'volume24hFiat',
         display: { base: 'none', lg: 'table-cell' },
-        Cell: ({ row }: { value: string | undefined; row: RowProps }) => {
-          const pool = row.original
-
-          const { data: swapsData } = useQuery({
-            ...reactQueries.midgard.swapsData(pool.asset, 'hour', 7 * 24),
-            select: selectSwapsData,
-          })
-
-          const volumeStats = useMemo(() => {
-            if (!swapsData) return
-            return getVolumeStats(swapsData, runeMarketData.price)
-          }, [swapsData])
-
-          return (
-            <Skeleton isLoaded={!!volumeStats}>
-              <Skeleton isLoaded={!!volumeStats} display={mobileDisplay}>
-                <Amount.Fiat value={volumeStats?.volume24hFiat ?? '0'} />
-              </Skeleton>
+        Cell: ({ value: volume24hFiat }: { value: string | undefined; row: RowProps }) => (
```

Comment:
should we name these volume values to indicate they are fiat precision?

Reply:
If you mean user-currency vernacular, we may want to do one more pass across the board, but in this specific case this is just re-exposing the naming of the original prop as-is!